api/src/middlewares/requestRateLimiter.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from "express";
+import { RateLimiterRedis } from "rate-limiter-flexible";
+import { redisClient } from "../database/config/redisClient";
+
+const rateLimiter = new RateLimiterRedis({
+    storeClient: redisClient,
+    keyPrefix: 'middleware',
+    points: 10,
+    duration: 1,
+    useRedisPackage: true
+});
+
+const rateLimterMiddleware = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Typo in `Limter`.

Reply:
Thanks. Commit: 5407ff0a58f6e756de0c895c64d6b62931bcea9a